pub/js/options.js

```diff
@@ -251,58 +255,52 @@ function w3tc_starts_with(s, starts_with) {
+		},
+		browsercache_security_pkp_extra: {
+			maxage: 'The time, in seconds (as defined under the "Expires Header Lifetime" box of "Media & Other Files"), that the browser should remember that this site is only to be accessed using one of the defined keys. This only affects the site\'s main domain.',
+			maxageinc: 'The time, in seconds (as defined under the "Expires Header Lifetime" box of "Media & Other Files"), that the browser should remember that this site is only to be accessed using one of the defined keys. This affects the site\'s subdomains as well.'
+		},
+		browsercache_security_pkp_report_only: {
+			0: 'This instructs the browser to enforce the <acronym title="HTTP Public Key Pinning">HPKP</acronym> policy.',
+			1: 'This sets up <acronym title="HTTP Public Key Pinning">HPKP</acronym> without enforcement allowing you to use pinning to test its impact without the risk of a failed connection caused by your site being unreachable or <acronym title="HTTP Public Key Pinning">HPKP</acronym> being misconfigured.'
+		}
+	};
 
-	jQuery('#browsercache_security_hsts_directive,#browsercache_security_xfo_directive,#browsercache_security_xss_directive,#browsercache_security_pkp_extra,#browsercache_security_pkp_report_only').on( 'change',
-	function() {
-		jQuery('#' + jQuery(this).attr('id') + '_description').html('<i>' + directive_description[jQuery(this).attr('id')][jQuery(this).val()] + '</i>');
+	jQuery('#browsercache_security_hsts_directive,#browsercache_security_xfo_directive,#browsercache_security_xss_directive,#browsercache_security_pkp_extra,#browsercache_security_pkp_report_only').on('change',
+		function() {
+			jQuery('#' + jQuery(this).attr('id') + '_description').html('<i>' + directive_description[jQuery(this).attr('id')][jQuery(this).val()] + '</i>');
 			if (jQuery(this).attr('id') == 'browsercache_security_xfo_directive') {
 				if (jQuery(this).val() == 'allow') {
 					jQuery('#browsercache_security_xfo_allow').show();
-				}else {
+				} else {
 					jQuery('#browsercache_security_xfo_allow').hide();
 				}
 			}
-	});
+		});
 
-	if(jQuery('#browsercache_security_xfo_allow').length) {
+	if (jQuery('#browsercache_security_xfo_allow').length) {
 		if (jQuery('#browsercache_security_xfo_directive').val() == 'allow') {
 			jQuery('#browsercache_security_xfo_allow').show();
 		} else {
 			jQuery('#browsercache_security_xfo_allow').hide();
 		}
-		jQuery('#browsercache_security_hsts_directive,#browsercache_security_xfo_directive,#browsercache_security_xss_directive,#browsercache_security_pkp_extra,#browsercache_security_pkp_report_only').on( 'change', );
+		jQuery('#browsercache_security_hsts_directive,#browsercache_security_xfo_directive,#browsercache_security_xss_directive,#browsercache_security_pkp_extra,#browsercache_security_pkp_report_only').on('change', );
```

Comment:
Missing callback function.  If this is a duplicate of line 285, then just remove this line.

Reply:
Line removed